### ISSUE TITLE
Planks Removed from Methanol Recipe in #61452?

### ISSUE THREAD

**Issue by ChromePoptart** [CleverRaven/Cataclysm-DDA#62495]
### Describe the bug

#61452 adjusted the recipe for crafting methanol from planks or splintered wood:
```
-    "components": [ [ [ "splinter", 40 ], [ "2x4", 12 ] ], [ [ "water", 3 ], [ "water_clean", 3 ] ] ]
+    "components": [ [ [ "splinter", 55 ] ], [ [ "water", 50 ], [ "water_clean", 50 ] ] ]
```
Was it intentional to fully remove the ability to craft methanol using planks?  In "describe the solution", @jim-huynh references changing the number of required planks from 12 to 2 (which I would think would also imply decreasing the number of splintered wood required to 7 or 8?).  I didn't see any further discussion related to removing planks as a potential component in the discussion in #61452, but maybe discussion was held elsewhere.

Was it intentional to remove planks as an option, leaving only splintered wood?

### Steps to reproduce

![image](https://user-images.githubusercontent.com/109529281/204839823-8d4c6922-2898-4ac3-b622-583b6f383319.png)

### Expected behavior

If it wasn't decided elsewhere to remove planks as a component option, I'd expect them to remain an option (albeit with lowered quantities required, per the goal of #61452.

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19044.2251 (21H2)
- Game Version: c804856 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Blaze Industries [blazeindustries],
    Bionic Slots [cbm_slots]
]

**Comment by Maleclypse** [comment]
Yes, because to make methanol you have to shred wood into wood chip or splinter sizes.  So it was either remove planks or increase tool qualities and time needed to represent shredding the wood into appropriate sizes. Then if someone already had splinters they'd need tools and time that was actually unnecessary for them so it's better to just remove that as an option and let the player shred the wood in a separate action unfortunately.

**Comment by ChromePoptart** [comment]
Thanks for the explanation and clarification, @Maleclypse!
